packages/firebase_ml/android/src/main/java/io/flutter/plugins/firebaseml/FirebaseMLPlugin.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.package io.flutter.plugins.firebaseml;
+
+package io.flutter.plugins.firebaseml;
+
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/** FirebaseMLPlugin */
+public class FirebaseMLPlugin implements FlutterPlugin, MethodCallHandler {
+
+  private static final String CHANNEL_NAME = "plugins.flutter.io/firebase_ml";
+
+  private MethodChannel channel;
+
+  public static void registerWith(Registrar registrar) {
```

Comment:
javadoc is required for this method.

https://screenshot.googleplex.com/D0JkcetFxiB

I can't share our whole guide, so here is a link to external practices:
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html

Reply:
Done.